docs/gaia-tutorials/deploy-testnet.md

```diff
@@ -41,18 +41,18 @@ This guide helps you create a single validator node that runs a network locally
 cd $HOME
 
 # Initialize the genesis.json file that will help you to bootstrap the network
-gaiad init --chain-id=testing testing
+gaiad init my-node --chain-id my-chain
 
 # Create a key to hold your validator account
-gaiacli keys add validator
+gaiad keys add my-account
 
 # Add that key into the genesis.app_state.accounts array in the genesis file
 # NOTE: this command lets you set the number of coins. Make sure this account has some coins
 # with the genesis.app_state.staking.params.bond_denom denom, the default is staking
-gaiad add-genesis-account $(gaiacli keys show validator -a) 1000000000stake,1000000000validatortoken
+gaiad add-genesis-account $(gaiad keys show my-account -a) 1000000000stake,1000000000validatortoken
 
 # Generate the transaction that creates your validator
-gaiad gentx --name validator
+gaiad gentx my-account 1000000000stake --chain-id my-chain
```

Comment:
```suggestion
gaiad gentx my-account --chain-id=my-chain 1000000000stake
```

Reply:
Works with `gaiad gentx my-account 1000000000stake --chain-id my-chain`. I'm going with this one because the CLI parameters show arguments need to be right after each other, and there's typically no equal sign for assigning a value to a parameter (chain-id in this case).